manager/integration/tests/test_basic.py

```diff
@@ -168,6 +193,10 @@ def test_volume_iscsi_basic(clients, volume_name):  # NOQA
     assert volumes[0]["frontend"] == "iscsi"
     assert volumes[0]["endpoint"].startswith("iscsi://")
 
+    dev = iscsi_login(volumes[0]["endpoint"])
```

Comment:
Use `try...finally` in case there is an exception happened when iSCSI initiator connected.